openquake/commonlib/export/hazard.py

```diff
@@ -265,28 +265,29 @@ def add_imt(fname, imt):
     return os.path.join(os.path.dirname(fname), newname)
 
 
-def export_hcurves_by_imt_csv(key, dest, sitecol, curves_by_imt,
-                              imtls, investigation_time=None):
+def export_hcurves_by_imt_csv(key, kind, rlzs_assoc, fname, sitecol,
+                              curves_by_imt, oq):
     """
     Export the curves of the given realization into CSV.
 
     :param key: output_type and export_type
     :param dest: name of the exported file
     :param sitecol: site collection
```

Comment:
Please add to the docstring also the parameters `kind`, `rlzs_assoc` and `fname` and remove `dest`